recipes/viennarna/remove-register.patch

```diff
@@ -0,0 +1,64 @@
+diff --git a/src/RNAlocmin/hash_util.h b/src/RNAlocmin/hash_util.h
+index e5dcfd3..50a0138 100644
+--- a/src/RNAlocmin/hash_util.h
++++ b/src/RNAlocmin/hash_util.h
+@@ -79,10 +79,10 @@ struct hash_eq {
+ struct hash_fncts{
+   size_t operator()(const Structure &x) const {
+ 
+-  register short *k;        /* the key */
```

Comment:
Might be worth also opening an issue or PR upstream for this so we can remove the patch at some point.

Reply:
:+1: Thanks for the suggestion, I plan to do this.